v23.1/transaction-retry-error-reference.md

```diff
@@ -63,27 +101,38 @@ This "conversational" design means that there is no way for the server to always
 - [ABORT_REASON_TIMESTAMP_CACHE_REJECTED](#abort_reason_timestamp_cache_rejected)
 - [injected by `inject_retry_errors_enabled` session variable](#injected-by-inject_retry_errors_enabled-session-variable)
 
+Each transaction retry error listed includes an example error as it would appear from the context of the client, a description of the circumstances that cause that error, and specific guidance for addressing the error.
+
 ### RETRY_WRITE_TOO_OLD
 
 ```
 TransactionRetryWithProtoRefreshError: ... RETRY_WRITE_TOO_OLD ...
 ```
 
+_Error type_: Serialization error
```

Comment:
I'm definitely not in love with the strange formatting with italics for these subsection headers?